src/main/frontend/format/mldoc.cljs

```diff
@@ -178,3 +190,44 @@
                      (if (and properties (seq properties))
                        properties))]
     (into {} properties)))
+
+
+(deftest test-parse-org-properties
```

Comment:
The testings look great! A small nitpick is that all the tests are separated from the main namespace, e.g. this function can move to the file `web/src/test/frontend/format/mldoc_test.cljs`.

Reply:
done!

Reply:
Great! Thank you for the PR and tests!

Reply:
@karlicoss It's on logseq.com now!

Reply:
exciting! Thanks for assisting!